src/config.json

```diff
@@ -1190,6 +1190,61 @@
     "nftinit.com"
   ],
   "blacklist": [
+    "dappconnectpad.com",
+    "exchange-otc.info",
+    "app.step-launchpad.com",
+    "airdrop-trustwallet.io",
+    "activatewallet.org",
+    "macimprofit.com",
+    "securencrypt.netlify.app",
+    "metamaskrestore.vercel.app",
+    "boardapesyachtclub.com",
+    "stepnnft.online",
+    "facexnft.com",
+    "smartentries.org",
+    "smartvalidation.online",
+    "monkeykingdom.store",
+    "walletreactivate.org",
+    "metaverse-adidas.net",
+    "jumpn.org",
```

Comment:
Please can you remove `jumpn.org`, it seems to be valid 

https://twitter.com/JUMPNofficial/status/1527190440520617985